src/box/txn.h

```diff
@@ -525,6 +525,10 @@ struct txn {
 	 * Nesting level of space on_replace triggers for current txn.
 	 */
 	int space_on_replace_triggers_depth;
+	/**
+	 * Whether the transaction has temporary statements.
+	 */
+	bool has_temporary_stmts;
```

Comment:
Let's elaborate in description why we need the flag at all (rows of temporary statements are erased).

Reply:
Done.